packages/web-components/fast-components/src/picker/index.ts

```diff
@@ -118,8 +94,4 @@ export const fastPickerListItem = PickerListItem.compose({
     styles: pickerListItemStyles,
 });
 
-/**
- * Styles for PickerListItem
- * @public
- */
-export const PickerListItemStyles = pickerListItemStyles;
+export { pickerStyles, pickerListItemStyles, pickerMenuOptionStyles, pickerMenuStyles };
```

Comment:
*Technically* this is a break. These are functions and should be exported as such. Picker itself is in alpha, but these were marked public. I'm happy to add deprecated versions of those removed here.

Reply:
Probably not an issue. I don't think picker is documented anywhere, so likely folks don't really know this exists.

Reply:
My thought as well 👍